pkgs/games/fteqw/default.nix

```diff
@@ -0,0 +1,137 @@
+{ lib
+, stdenv
+, fetchsvn
+, runCommand
+, gzip
+, libvorbis
+, libmad
+, SDL2
+, SDL2_mixer
+, libpng
+, alsa-lib
+, gnutls
+, zlib
+, libjpeg
+, vulkan-loader
+, speex
+, libopus
+, xorg
+, vulkan-headers
+, libGL
+, pkg-config
+}:
+let
+  common = { pkgName, releaseFile, buildFlags, buildInputs, nativeBuildInputs, postFixup ? "", description }:
+  stdenv.mkDerivation rec {
+    inherit buildFlags buildInputs nativeBuildInputs postFixup;
+    pname = pkgName;
+    version = "unstable-2022-08-09";
+    src = fetchsvn {
+      url = "https://svn.code.sf.net/p/fteqw/code/trunk";
+      rev = "6303";
+      sha256 = "sha256-tSTFX59iVUvndPRdREayKpkQ+YCYKCMQe2PXZfnTgPQ=";
+    };
+    makeFlags = [
+      "PKGCONFIG=pkg-config"
```

Comment:
That is probably not cross friendly but I don't know the right syntax to be cross friendly right now.

Reply:
We export `PKG_CONFIG` as the pkg-config executable for the host platform, so it should just be `PKGCONFIG=$(PKG_CONFIG)`.

Reply:
I'll update the PKGCONFIG variable, but I can't test on other platforms to ensure it works elsewhere. The codebase should portably work on Windows and macOS, but I have neither to test on.

Reply:
This is more for pkgsCross.aarch64-multiplatform.fteqw